src/web/components/SiteTour/tourHelpers.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('testing tour storage helper functions', () => {
     mockUser.currentParticipantUserRoles = [{ id: UserRoleId.Admin, roleName: 'Admin' }];
     mockUser.participants = [mockParticipant, mockParticipant2] as Participant[];
     mockParticipant.currentUserRoleIds = [UserRoleId.Admin];
-    localStorage.removeItem(tourStorageKey);
+    localStorage.setItem(tourStorageKey, JSON.stringify({ seenForVersions: ['0.1.0'] }));
 
```

Comment:
Setting it to an early version so it shows all the steps are present - can't leave it empty or it won't show any tour steps